### ISSUE TITLE
We aren't getting Google Analytics

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/righttoknow#852]
We aren't getting any data from Google analytics because we are still using the old UA property instead of the new GA4 property.

To do:
- [ ] Confirm Alaveteli can support GA4 on our current version.
- [ ] If it can, update the code in infrastructure project

**Comment by katska** [comment]
@simonzippy is this a specific task that might be up your alley? 
@mlandauer has implemented analytics for Planning Alerts, so if there are OAF specific ways of approaching this, maybe worth a look.

**Comment by benrfairless** [comment]
Now we have Plausible let's switch to that.

**Comment by Br3nda** [comment]
Do we want GA as well as plausible?

**Comment by benrfairless** [comment]
> Do we want GA as well as plausible?
> 

I don't think so, unless it's needed for advertising tracking - @katska should be able to tell us if that's the case.

I can see that according to my iPhone the site is still trying to connect to Google analytics.

**Comment by katska** [comment]
@benrfairless no. Let's just stick with Plausible.io 
As you point out Google Analytics is dead now, and so completely redundant. We wont set up GA4, we do now have Plausible instead. Closing this issue, if there's any problem with Plausible from the switch let's look at that separately in a fresh issue.

**Comment by simonzippy** [comment]
Detail on the new Google Analytics code and tracking tag we would need if we needed to reapply Google Analytics for Digital Ninjas to managed Google Adwords is [here >](https://github.com/openaustralia/infrastructure/pull/198#issuecomment-3203843141)

**Comment by katska** [comment]
I have now shared the plausible.io with Digital Ninjas and again no to GA4
